AboutMe.md

```diff
@@ -1,9 +1,8 @@
 ## About Me
-<p>Name:</p>
-<p>Preferred Name:</p>
-<p>Grade Level:</p>
-<p>Email:</p>
-<p>Computer Science Courses Taken:</p>
-<p>Computer Science Interest Areas:</p>
-<p>After High School Plans:</p>
-
+<p>Name: Trey</p>
+<p>Preferred Name: Trey</p>
+<p>Grade Level: 12</p>
+<p>Email:24tjohnston@adriansd.org</p>
+<p>Computer Science Courses Taken: Intro to computer science, Exploring Computer science, AP computer science</p>
+<p>Computer Science Interest Areas: AI, Codeing, Computer hardware </p>
+<p>After High School Plans: Go to college to study Computer Science </p>
```

Comment:
I'd love to help support these plans. My oldest son is starting his sophomore year at OSU Cascades in Bend, OR in Computer Science. We spent a lot of time comparing schools and cost. Also looked at coding bootcamp programs such as Flatiron, Bloomtech, and Coding Dojo. Let me know more about your plans and I'll do what I can to help.